CONTRIBUTING.md

```diff
@@ -174,40 +157,31 @@ applicable.
 Feel free to contribute standalone pull requests to add new tests or improve
 existing tests as well.
 
-See [Unit testing](https://docs.godotengine.org/en/latest/contributing/development/core_and_modules/unit_testing.html)
-for information on writing tests in Godot's C++ codebase.
+See [Unit testing](https://docs.blazium.app/en/latest/contributing/development/core_and_modules/unit_testing.html)
```

Comment:
I don't think we have an /en/ for languages yet

Reply:
Fixed, thank you!